app/code/Magento/UrlRewriteGraphQl/Model/Resolver/AbstractEntityUrl.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Copyright 2025 Adobe
```

Comment:
It should be year of file creation
```suggestion
 * Copyright 2021 Adobe
```